### DISCUSSION TITLE
How to add special Function Keys (i.e. F13...) in Advanced Panel

### DISCUSSION THREAD

**Discussion by bauersimon** [AntiMicroX/antimicrox#817] [Q&A] [answered]
In the normal key mapping pannel, one can select special function keys (i.e. `F13`) from the dropdown on the top right. However, if I want to use them for Advanced things, there is no way of selecting them (despite for the first action, but I need it later on, combined with `F14`).

I was able to somewhat get it working a few months ago by assigning the `F13` to a controller button and then when the `Insert` action is waiting for a key-press, press the controller to register `F13`. But it seems now all controller presses are ignored.

I guess I will manually edit the config file for now.

**Comment by pktiuk** [comment, marked as answer]
I think this is a missing button.   
Adding it is a good idea.   
In general, there are plans for remaking Advanced Buttons dialog: https://github.com/AntiMicroX/antimicrox/issues/284
